Spatial.md

```diff
@@ -226,7 +223,6 @@ dealing with open standard formats or proprietary formats.
 <!--Roger, I assume we can keep this information about rgrass7 for now and update it when rgrass is on CRAN (I just discovered the name change using a GitHub search)-->
 -   *GRASS GIS:* Integration with version 7.\* of the leading open source GIS,
     GRASS GIS, is provided in CRAN package `r pkg("rgrass7")`.
-    For GRASS 6.\*, use `r pkg("spgrass6")`.
 -   *SAGA GIS:* `r pkg("RSAGA")` and `r pkg("Rsagacmd")` offer shell-based
     wrapper for SAGA GIS commands.
 -   *Quantum GIS (QGIS):* QGIS2 was supported by RQGIS 
```

Comment:
For what it's worth, RQGIS2 is now archived on GitHub ( https://github.com/r-spatial/RQGIS ) and requires Python 2; I didn't remove this because the link still goes to something that someone _could_ install.